third_party/move/move-model/src/model.rs

```diff
@@ -637,11 +644,21 @@ impl GlobalEnv {
         self.generated_by_v2 = yes
     }
 
-    /// Returns if this is generated by v2.
+    /// Returns true if this is generated by v2.
     pub fn generated_by_v2(&self) -> bool {
```

Comment:
nit: rename to `is_generated_by_v2`, and remove the comment.

Reply:
We typically don't add the `is` qualifier.